src/Jank/Scorepads/__tests__/ScorepadList.test.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  fireEvent,
+  render,
+  waitForElementToBeRemoved,
+  within,
+} from '@testing-library/react';
+
+import ScorepadList from '../ScorepadList';
+import * as ApiService from '../../../services/api.service';
+
+const get = jest.spyOn(ApiService, 'get');
+
+describe('Scorepad', () => {
+  beforeEach(() => {
+    get.mockResolvedValue([]);
+  });
+
+  it('renders header by default', async () => {
+    const { findByText } = render(<ScorepadList />);
+    expect(await findByText('Bestehende Spiele:')).toBeInTheDocument();
+  });
+
+  it('renders scorepads received from the api', async () => {
+    get.mockResolvedValue([{ _id: '1', players: [{ _id: '12', name: 'Matteo', picture: '' }], created_at: '2021-02-07T16:22:55.000Z' }]);
+    const { findByText } = render(<ScorepadList />);
+    const name = await findByText('Matteo');
+    expect(name).toBeInTheDocument();
+    const date = await findByText('Erstellt am: 07.02.2021');
+    expect(date).toBeInTheDocument();
```

Comment:
It's fine like it is, but if you wanted it to be more concise (and consistent with the test above), you could just have it in one line each:
```
expect(await findByText('Matteo')).toBeInTheDocument();
expect(await findByText('Erstellt am: 07.02.2021')).toBeInTheDocument();
```
```